apps/gdalalg_raster_calc.cpp

```diff
@@ -572,6 +572,19 @@ GDALRasterCalcAlgorithm::GDALRasterCalcAlgorithm() noexcept
         .SetRequired()
         .SetPackedValuesAllowed(false)
         .SetMinCount(1);
+
+    AddArg("skip-check-expression", 0,
```

Comment:
no-check-expression for consistency with no-check-extent and no-check-crs?

Reply:
OK. Note this is a hidden option just for unit test purposes for now. I'll add a code comment